react-bootstrap/index.d.ts

```diff
@@ -590,6 +590,7 @@ declare namespace ReactBootstrap {
         active?: boolean;
         noCaret?: boolean;
         eventKey?: any;
+        title?: any; // TODO: Add more specific type
```

Comment:
Yep, I've tried it but I have no idea why it doesn't accept the code that works ;)
`<NavDropdown eventKey={3} title={<i className="glyphicon glyphicon-user">Dropdown</i>} id='basic-nav-dropdown'>`
and I recieve compilation error `"Type Element is not assignable to Type 'Element' is not assignable to type 'string | (number & string) | (true & string) | (false & string) | ({} & string) | (ReactElement<a...'.`